src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -338,6 +338,16 @@
       gridColumns: 3,
       readPermissionRequired: true
     },
+    {
+      class: 'Reference',
+      of: 'foam.nanos.auth.User',
+      name: 'createdFor',
+      includeInDigest: true,
+      section: 'approvalRequestInformation',
+      order: 116,
+      gridColumns: 3,
+      readPermissionRequired: true
```

Comment:
```suggestion
```

Reply:
Because we are planning to display createdFor to the user to show what business the request came from instead of doing the BusinessApprovalRequest stuff.

Reply:
updated.